browserexport/browsers/common.py

```diff
@@ -117,7 +119,8 @@ def handle_glob(bases: Sequence[Path], stem: str, recursive: bool = False) -> Pa
     recur_desc = "recursive" if recursive else "non recursive"
     logger.debug(f"Glob {bases} with {stem} ({recur_desc}) matched {dbs}")
     if len(dbs) > 1:
-        raise RuntimeError(errmsg.format(dbs))
+        human_readable_db_paths: str = "\n".join([str(db) for db in dbs])
+        raise RuntimeError(errmsg.format(human_readable_db_paths))
```

Comment:
The error message looks something like this now:

```
RuntimeError: Expected to match a single database, but found:
/var/home/apatel/.var/app/com.brave.Browser/config/BraveSoftware/Brave-Browser/Default/History
/var/home/apatel/.var/app/com.brave.Browser/config/BraveSoftware/Brave-Browser/Guest Profile/History
/var/home/apatel/.var/app/com.brave.Browser/config/BraveSoftware/Brave-Browser/System Profile/History

You can use the --profile argument to select one of the profiles/match a particular file
```
(the difference being that the paths are printed on new lines, which makes them a little easier to read)

Reply:
yep looks good, thanks